modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -623,8 +623,11 @@ private void SetDiagonal(Vector3 diagonal)
         /// </summary>
         /// <param name="target">The position to look at.</param>
         /// <param name="up">The relative up direction.</param>
+        /// <param name="useModelFront">If specified, then the model is oriented in reverse,
+        /// towards the model front axis (+Z, Vector3.MODEL_FRONT),
+        /// which is more useful for orienting 3D models.</param>
```

Comment:
- When the contents of a `<param>` spans multiple lines, I prefer to keep the tag in separate lines.
- Also, changed the constant to use the C# name (`Vector3.ModelFront`), we could use `<see cref="Vector3.ModelFront"/>`.

```suggestion
        /// <param name="useModelFront">
        /// If specified, then the model is oriented in reverse,
        /// towards the model front axis (+Z, Vector3.ModelFront),
        /// which is more useful for orienting 3D models.
        /// </param>
```